xamarin/Xamarin.Forms
# [Android, iOS] Allow transparent background color in SwipeView Content

### Description of Change ###

Allow transparent background color in SwipeView Content. The logic to set the SwipeView BackgroundColor is the following:
- If the SwipeView Content has a background color set, that color is used. If the color is transparent, it applies transparencies.
- If the SwipeView Content does not have a BackgroundColor applied and SwipeView does, that color is used.
- If the Content has no BackgroundColor, we set a background color based on system colors.

### Issues Resolved ### 

- fixes #9305

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
<img width="357" alt="Captura de pantalla 2020-02-26 a las 16 45 52" src="https://user-images.githubusercontent.com/6755973/75361513-b197da80-58b7-11ea-9fb8-e1a1b274cc76.png">

_NOTE: The second SwipeView have a Transparent BackgroundColor._

### Testing Procedure ###
Launch Core Gallery and navigate to the Issue 9305. Verify that can have a SwipeView with a transparent BackgroundColor.

### PR Checklist ###
